index.js

```diff
@@ -1,136 +0,0 @@
-// import Readline
-const readline = require('readline')
-const rl = readline.createInterface(process.stdin, process.stdout)
-
-function ask(questionText) {
-     return new Promise((resolve, reject) => {
-          rl.question(questionText, resolve)
-     })
-}
-
-// GLOBAL VARIABLES
-let hiNum = 0
-let loNum = 0
-let secretNumber = 0
-let currentGuess = 0
-let guessLowerThan = 0
-let guessHigherThan = 0
-let guessConfirm = "N"
-let hiLo = ""
-let numOfGuesses = 0
-
-// GLOBALFUNCTIONS
-
-function randomInteger(min, max) {
-     let range = max - min + 1;
-     return +(min + Math.floor(Math.random() * range));
-}
-
-function newGuess(max, min) {
-     return Math.floor(((max - min + 1) / 2) + min)
-}
-
-function cheatChecker(cheatCode) {
-     if (cheatCode === 1) { // check for number out of range or alpha character
-          if (secretNumber < 1 || secretNumber > hiNum) {
-               console.log("\nYou are trying to cheat.\nNext time pick a number in range.")
-               process.exit()
-          } else if (isNaN(secretNumber)) {
-               console.log("\nLetters are not allowed.\nNext time pick a NUMBER in range.")
-               process.exit()
-          }
-     } else if (cheatCode === 2) { // check if human lied about number beeing higher
-          if (guessLowerThan - 1 === currentGuess) {
-               console.log("\nStupid Human\nYou already said it was lower than " + guessLowerThan + ".\nSo it cannot also be higher than " + currentGuess + ".")
-               process.exit()
-          }
-     } else if (cheatCode === 3) {  // check if human lied about number beeing lower
-          if (guessHigherThan + 1 === currentGuess) {
-               console.log("\nStupid Human\nYou already said it was higher than " + guessHigherThan + ".\nSo it cannot also be lower than " + currentGuess + ".")
-               process.exit()
-          }
-     }
-}
-
-start()
-
-// Start game and choose role of computer and human - who picks number and who guesses - launch game version
-async function start() {
-     console.log("\nLet's play a game where one of us picks a number inside of a range\nand the other tries to guess it.\n")
-     gameVersion = await ask("Do you want to be the number picker or number guesser?\nEnter P for picker\nEnter G for guesser\n>")
-     if (gameVersion === 'P') {
-          computerGuesses()
-     } else if (gameVersion === 'G') {
-          humanGuesses()
-     } else {
-          console.log("Pick capital P or capital G only.\nPlease start over.")
-          process.exit()
-     }
-}
```

Comment:
doing a `if(gameVersion.toUpperCase() === 'P')` could make this a little more user friendly

also putting this within a loop and removing the process.exit() would make this more user friendly as well, but good work overall on having that catch in there to not just break the game if an incorrect input was given